state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -1571,3 +1581,21 @@ impl<T: Config<I>, I: 'static> DepositApi<T::TargetChain> for Pallet<T, I> {
 		))
 	}
 }
+
+impl<T: Config<I>, I: 'static> TransactionFeeApi<T::TargetChain> for Pallet<T, I> {
+	fn accrue_transaction_fee(
+		_asset: <T::TargetChain as Chain>::ChainAsset,
```

Comment:
Is asset isn't actually used, can it be removed?

Reply:
It is needed because I use the same trait for the handler. See `impl_transaction_fee_api_for_anychain`. Not perfect, but still better than adding another trait I think.